auto_round/autoround.py

```diff
@@ -1189,6 +1197,21 @@ def quantize_rtn(self) -> tuple[torch.nn.Module, dict[str, Any]]:
 
         all_to_quantized_module_names: list[str] = [n for n, m in self.model.named_modules() if check_to_quantized(m)]
 
+        if is_nv_fp(self.data_type):
+            from auto_round.data_type.nvfp import calculate_gparam
+            from auto_round.data_type.utils import update_fused_layer_global_scales
+
+            pbar = tqdm(all_to_quantized_module_names)
+            for name in pbar:
+                pbar.set_description(f"Calculate weight global scale: {name}")
+                m = get_module(self.model, name)
```

Comment:
they are too many specific codes for nvfp, it would better to move the code from autoround.py to other places to make the code clean